covid-policy-pybay-2023

polkapolka opened the pull request.

polkapolka committed:
covid-policy-pybay-2023

moshez commented on CovidPolicy.md:
```diff
@@ -0,0 +1,43 @@
+
+**At a minimum, we will be following all mandated venue, municipality, and province guidelines. This list, and all protocols, are subject to change if the COVID-19 situation changes.**
+
+### Vaccine Expectation
```
I'm not comfortable with the wording "if you are asked". Can we either commit to ask everyone for proof of vaccination, or explicitly note that vaccination will not be checked, but highly recommended?

polkapolka replied:
That wording is because SF may change it's covid policy to require it, and at that time we would check it.

moshez commented on CovidPolicy.md:
```diff
@@ -0,0 +1,43 @@
+
+The proof of vaccination record must show the name of the vaccinated individual along with one other personal identifier (such as date of birth or passport number), name of the manufacturer for vaccine received, and date(s) of vaccination. It must also include the name of the official source issuing the record (e.g., public health agency, government agency, or other authorized vaccine provider).
+
+### Negative COVID-19 Test Expectation
```
If we are not requiring vaccination, having something in lieu of that seems also weird. It would be nice if we could commit to, and mention, that we will have some testing kits for people to self-administer before they enter the venue on a voluntary basis.

moshez commented on CovidPolicy.md:
```diff
@@ -0,0 +1,43 @@
+
+In line with San Francisco guidelines, we prioritize the well-being of our attendees at this outdoor event. While masks and vaccinations are not currently mandated, we reserve the right to change this at any time to comply with San Francisco guidelines. Refunds will be offered for attendees who are unable to comply with updated guidelines. At this time, personal choices regarding their usage are  ...
+
+We will have masks available at the conference for anyone who needs them. It is also expected that requests for personal space will be respected by attendees and are covered under the code of conduct. 
```
Maybe we can break this into two parts?

"It is expected that requests for personal space will be respected by attendees. Respecting such requests is covered by the PyBay code of conduct."

polkapolka replied:
agreed.

moshez commented on CovidPolicy.md:
```diff
@@ -0,0 +1,43 @@
+
+We will have masks available at the conference for anyone who needs them. It is also expected that requests for personal space will be respected by attendees and are covered under the code of conduct. 
+
+While we follow San Francisco's guidance for this outdoor events and don't require masks or proof of vaccination, we respect individual choices. Please prioritize hygiene, personal space, and your health. Stay home if unwell. Your safety matters most as we enjoy the event together.
```
Either "guidance for outdoor events" or "guidance for this outdoor event"...

polkapolka replied:
agreed.

moshez reviewed (commented):
I'm not comfortable approving the policy as written -- I have pointed out where it lacks clarity. I agree with the spirit of the policy.

polkapolka committed:
Update CovidPolicy.md

response to comments

jamesabel commented:
"Vaccine Expectation" and below are *if* the policy changes, correct? If so, it seems confusing to have them listed now. To me, it seems like the "Expectation" sections hint that it's a current expectation, since it's not explicitly saying it may become an expectation at a later date. And, since we're talking about a future requirement, we don't necessarily know what that'll be.

mahmoud commented on CovidPolicy.md:
```diff
@@ -0,0 +1,43 @@
+
+### Vaccine Expectation
+
+If you are asked to show proof of vaccination to attend in person, you will need to show that you are ‘fully vaccinated’ against the COVID-19 virus. We expect that you:
```
```suggestion
Should local regulations change to require showing vaccination, we expect that you:
```

polkapolka replied:
Changed

mahmoud commented on CovidPolicy.md:
```diff
@@ -0,0 +1,43 @@
+
+*     have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
+*     that your most recent dose was at least fourteen days before the conference start
+*     that your most recent dose or booster was no more than twelve months before the conference start
```
```suggestion
```

Did local regulations ever include this? If not, I suggest removing it.

polkapolka replied:
changed

mahmoud commented on CovidPolicy.md:
```diff
@@ -0,0 +1,43 @@
+
+### Negative COVID-19 Test Expectation
+
+In lieu of proof of vaccination, participants may instead show proof of a negative COVID-19 test administered by a verified provider to enter the event.
+
+*     Provide a negative COVID-19 test upon entry to the event for the first time.
+*     The test must be a PCR, NAAT, or Antigen test taken within 24 hours of picking up your badge on-site.
+*     The test may NOT be an unproctored at-home test. Tests must be administered by a verified provider, either locally in Vancouver or virtually before you travel from your home city (if time permits).
+*     The test results must include your name and the date and time of the test.
```
```suggestion
We recommend taking a COVID test before attending PyBay. In the event of a positive test result, we offer attendees a refund and ask that they stay home. Additionally, we will have a small supply of tests on hand, should attendees prefer to test on-site.
```

polkapolka replied:
changed

mahmoud reviewed (commented):
Headings may need a bit of a tweak, but here's a rough cut with suggestions that convey what we discussed at the board meeting.

polkapolka committed:
Update and rename CovidPolicy.md to 2023-pybay-covid-policy.md

update in response to comments

kilodalton commented on 2023-pybay-covid-policy.md:
```diff
@@ -0,0 +1,37 @@
+
+Should local regulations change to require showing vaccination, we expect that you:
+
+* have completed a full round of vaccine (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
```
```suggestion
* are fully vaccinated for COVID-19 (1 dose of a single-dose vaccine or 2 doses of a two-dose vaccine)
```

kilodalton reviewed (approved):
I would prefer if the wording change for that one item to match common language on this subject, but otherwise approved.

polkapolka committed:
Update 2023-pybay-covid-policy.md

Co-authored-by: Karen Dalton <kd@karend.net>

kilodalton commented:
Thanks for the change, looks good.

jamesabel committed:
Update 2023-pybay-covid-policy.md

Small but (I think) important wording change that we encourage masks and vaccinations.

polkapolka committed:
Merge pull request #4 from BAPyA/jamesabel-covid-patch-1

Update 2023-pybay-covid-policy.md

jamesabel reviewed (approved):
LGTM

mahmoud reviewed (approved):
LGTM, great job integrating those changes.
